Mathlib/LinearAlgebra/Matrix/Symmetric.lean

```diff
@@ -40,7 +40,7 @@ theorem IsSymm.ext_iff {A : Matrix n n α} : A.IsSymm ↔ ∀ i j, A j i = A i j
   Matrix.ext_iff.symm
 
 /-- A version of `Matrix.ext` that unfolds the `Matrix.transpose`. -/
--- @[ext] -- Porting note: removed attribute
+-- @[ext] -- Porting note (https://github.com/leanprover-community/mathlib4/issues/11041): incorrect `ext`, not a structure or a lemma proving `x = y`.
```

Comment:
I'm not sure that anyone would think of using the `ext` tactic for a goal of the shape `A.IsSymm`. Maybe remove the attribute altogether?
```suggestion
```